packages/app/templates/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
Hmm, I couldn't find the `title` tag generated. Is it ok? Do the e2e tests work fine (e.g. `sar-window-check.e2e.ts`)? 

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/41268731/174599209-accf3a67-d1f0-4930-9db0-ef4b70fc65cf.png">

Reply:
Yes, I know. But, it's works in e2e test

Reply:
😮 

Just curious, do you know why? Previously, when I was simply setting the `title` tag in `BrowserWindow`, it was not working. Maybe the reason is that the title tag is not generated at all? 

<img width="383" alt="image" src="https://user-images.githubusercontent.com/41268731/174601219-8b1860f1-fc2d-449c-b325-f63ac7cdada5.png">

Reply:
Yep, it was overridden [here](https://github.com/mudita/mudita-center/pull/787/files#diff-8b54cdec43a0d5fe62f1f0022b422d3ba33eba5b41dd63638736e7ce7fed12abL42) by HTMLPlugin which put an empty `title` tag to HTML. That's why you should manually set the title to each page. But now, the electron automatically coupe with it.